webdriverio/index.d.ts

```diff
@@ -352,36 +352,37 @@ declare namespace WebdriverIO {
         ): Client<P>;
         getHTML<P>(includeSelectorTag: boolean): Client<P>;
 
-        getLocation(selector?: string): Size;
-        getLocation(selector: string, axis: string): number;
-        getLocation(axis: string): number;
+        getLocation(selector?: string): Location;
+        getLocation(selector: string, axis: Axis): number;
+        getLocation(axis: Axis): number;
```

Comment:
Yeah, I suppose that having the `Axis`-taking overload first would be the most appropriate thing, but long-term there should really be a distinct type with these methods that take an `Axis`. I won't bog you down on this PR, but it'd be ideal if you ever get the chance.